### DISCUSSION TITLE
[Bug] DocFx: Command not found on MacOS Sonoma

### DISCUSSION THREAD

**Discussion by markjamesm** [dotnet/docfx#9498] [General]
**Describe the bug**
I installed DocFx in the terminal, but attempting to build a project after install DocFx results in `zsh: command not found: docfx`. Running `dotnet tool update -g docfx` results in the following output:

```
Skipping NuGet package signature verification.
Tool 'docfx' was reinstalled with the latest stable version (version '2.74.1').
```

**To Reproduce**
1. Run `dotnet tool update -g docfx`
2. Attempt to build project `docfx docfx.json --serve`

**Expected behavior**
The solution builds.

**Context (please complete the following information):**
- OS: MacOS Sonoma 14.1.1, Intel Mac
- Docfx version: Latest

**Comment by yufeih** [comment]
Is it a transient issue? I tried the command on Intel Mac Sonoma 14.1.1 but failed to repro.

**Comment by DellLin** [comment]
I hava same issue.
```
> dotnet tool update -g docfx
Tool 'docfx' was reinstalled with the latest stable version (version '2.74.1').
```
```
> docfx init
zsh: command not found: docfx
```
Context (please complete the following information):

- OS: MacOS Sonoma 14.1.2, Apple M2
- Docfx version: Latest

I can't find the docfx path to set it in the zsh environment.

**Comment by filzrev** [comment]
It seems to be caused by problems with the .NET SDK Installer for mac.
- `https://github.com/dotnet/sdk/issues/22214`
- `https://github.com/dotnet/sdk/issues/22588`

It'll be resolved by adding `PATH` for `.NET Global Tools` manually.

**Comment by manwithsteelnerves** [comment]
This issue ate some hours of my time. Hope others find this use-ful. Thanks to @filzrev for the direction.

DocFx Installation

> Don't use homebrew as its not complete install setup currently.

```bash
dotnet tool update -g docfx
```

In zsh shell, type below

```bash
export PATH="$HOME/.dotnet/tools:$PATH"
```

> The above is required because the dotnet mac installer is setting ~ in $PATH which isn't expanding properly in zsh shell leading to command not found.
